Cython/Compiler/Main.py

```diff
@@ -716,7 +716,12 @@ def main(command_line = 0):
     args = sys.argv[1:]
     any_failures = 0
     if command_line:
-        options, sources = parse_command_line(args)
+        try:
+            options, sources = parse_command_line(args)
+        except IOError as e:
+            # TODO: IOError can be replaced with FileNotFoundError in Cython 3.1
+            print("{}: No such file or directory: '{}'".format(sys.argv[0], e.filename), file=sys.stderr)
```

Comment:
The assumption that this is due to a missing file seems a bit broad after the exception class change. Would be better to also look at `errno` first. And maybe just re-raise if it's something else.

Reply:
done